ci/authn-k8s/dev/policies/policy.template.yml

```diff
@@ -198,6 +203,31 @@
   role: !group conjur/authn-k8s/minikube/clients
   member: !layer some-policy
 
+# This policy is for testing that in case a host doesn't have any annotations defined
+# then the "authentication-container-name" annotation is defaulted to "authenticator"
+# and the host is authenticated.
+# We have a separate policy for this host so we can use the id '{{CONJUR_AUTHN_K8S_TEST_NAMESPACE}}/*/*'
+# without duplication.
+- !policy
+  id: host-without-container-name
+  body:
+    - !layer
+
+    - &hosts
```

Comment:
Build is currently failing with:

```
[2020-02-04T18:35:44.294Z] {"error":{"code":"validation_failed","message":"policy_text Error at line 221, column 5 in  : Duplicate anchor hosts","details":[{"code":"validation_failed","target":"policy_text","message":"Error at line 221, column 5 in  : Duplicate anchor hosts"}]}}
```

I think you have to rename this `&hosts` anchor. Maybe change it to `&host-without-container-name-hosts`. If you look at the policy above, it does a similar thing.

Reply:
correct.